json-parser/src/test/java/org/uniprot/core/json/parser/uniparc/UniParcCrossRefJsonConfigTest.java

```diff
@@ -9,17 +9,34 @@
 import org.uniprot.core.uniparc.UniParcCrossReference;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.uniprot.core.uniparc.UniParcEntry;
+import org.uniprot.core.uniparc.impl.UniParcCrossReferencePair;
+import org.uniprot.core.util.Pair;
+import org.uniprot.core.util.PairImpl;
+
+import java.util.ArrayList;
+import java.util.List;
 
 class UniParcCrossRefJsonConfigTest {
     @Test
     void test() {
-        UniParcCrossReference entry = getCompleteUniParcEntry().getUniParcCrossReferences().get(0);
+        UniParcEntry completeUniParcEntry = getCompleteUniParcEntry();
+        Pair<String, List<UniParcCrossReference>> entry = new UniParcCrossReferencePair(completeUniParcEntry.getUniParcId().getValue(), completeUniParcEntry.getUniParcCrossReferences());
+/*
+        ValidateJson.verifyJsonRoundTripParser(
+                UniParcCrossRefJsonConfig.getInstance().getFullObjectMapper(), completeUniParcEntry.getUniParcCrossReferences().get(0));
+
+        ValidateJson.verifyJsonRoundTripParser(
+                UniParcCrossRefJsonConfig.getInstance().getFullObjectMapper(), new ArrayList<>(completeUniParcEntry.getUniParcCrossReferences()));
+*/
+
         ValidateJson.verifyJsonRoundTripParser(
                 UniParcCrossRefJsonConfig.getInstance().getFullObjectMapper(), entry);
         ValidateJson.verifyEmptyFields(entry);
         try {
             ObjectMapper mapper = UniParcCrossRefJsonConfig.getInstance().getSimpleObjectMapper();
             String json = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(entry);
+            System.out.println(json);
```

Comment:
this println needs to be removed?

Reply:
We can remove it. But good for testing and visualising.